backend/api/Mapping/BaseMap.cs

```diff
@@ -9,13 +9,11 @@ public class BaseMap : IRegister
         public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<Entity.BaseEntity, Models.BaseModel>()
-                .IgnoreNullValues(true)
```

Comment:
what was the motivation for this change?

Reply:
A mistake!  Thanks for catching it!

Reply:
I've now added it to the global settings in the startup configuration.